### ISSUE TITLE
Vague error message (Can't resolve '') when importing package with invalid entry point

### ISSUE THREAD

**Issue by IjzerenHein** [expo/snack#79]
**Summary**

Importing `react-hooks-firebase` causes snackager to return error:

```
curl "https://snackager.expo.io/bundle/react-firebase-hooks?version_snackager=true&platforms=ios,android,web"
Can't resolve '' in '/tmp/snackager/snackager/buildStatus/1/react-firebase-hooks@2.2.0-ios,android,web/package'
```

react-firebase-hooks does have a main entry in its package.json but that refers to a non existent file. The error message returned by Snackager is very vague

**What platform(s) does this occur on?**

Android, iOS, Web

**SDK Version**

40

**Reproducible demo or steps to reproduce from a blank project**

curl "https://snackager.expo.io/bundle/react-firebase-hooks?version_snackager=true&platforms=ios,android,web"

**Comment by byCedric** [comment]
I don't think we can avoid this issue, but we can improve the error message. Is that a good way to solve this?

**Comment by IjzerenHein** [comment]
> I don't think we can avoid this issue, but we can improve the error message. Is that a good way to solve this?

Yep that sounds good enough

**Comment by howardwu1** [comment]
Is there a way to import these packages with invalid entry points? I don't think things like app.json work well on snack?
